### PR TITLE
travis: update config

### LINKED ISSUE #212: Travis CI build problems
In the travis `after_success` stage, we are using mprof, sfood / sfood-graph (from snakefood, sfood-graph package) and dot programs. These programs are not being installed by the install script.

For an example, please see a [sample log](https://travis-ci.org/prasadtalasila/IRCLogParser#L6995).

### PR DESCRIPTION
**Pull request by kaushik-rohit**
## What? Why?
Fix #212 

Changes proposed in this pull request:
- update email recipients
- comment sfood and mprof

## Checks
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure you have added required documentation
- [x] If you have refactored a code, make sure that you have compared the outputs pre and post refactoring. Add both pre and post refaoring screenshots below if the output is visual

## Notify reviewers
@rohangoel96

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@kaushik-rohit can you also address #219 with this PR?

**Comment by kaushik-rohit** [comment]
Sure, will do that.

**Comment by kaushik-rohit** [comment]
@prasadtalasila updated with changes required in #219

File: README.md [hunk 1]
```diff
@@ -13,37 +13,39 @@ The objective of this project is to utilize social network analysis techniques t
...
-A page on typical workflow for using IRCLogParser also exists on the documentation [here](http://prasadtalasila.github.io/IRCLogParser/TypicalUsage.html).
-
-Some of the visualisable sample outputs can be on the wiki, giving an idea about IRCLogParser's capabilites.
-
-
 ## Documentation
 
-IRCLogParser uses [Sphinx Python Documentation Genertor](http://www.sphinx-doc.org/en/stable/) for generating documentation of the library. The documentation is setup to work with [Google Style Docstrings](http://www.sphinx-doc.org/en/stable/ext/example_google.html) which eases the documentation writing process.
+IRCLogParser uses [Sphinx Python Documentation Generator](http://www.sphinx-doc.org/en/stable/) for generating documentation of the library.
+The documentation is auto-generated from the commits made on the [dev](https://github.com/prasadtalasila/IRCLogParser/tree/dev) branch. The documentation corresponding to the latest commit on dev branch is hosted here.
```
Comment (prasadtalasila) [inline README.md]: How about rephrasing

The documentation corresponding to the latest commit on dev branch is hosted here.

to 

Please see the [online documentation](prasad.talasila.in/IRCLogParser).

File: README.md [hunk 2]
```diff
@@ -13,37 +13,39 @@ The objective of this project is to utilize social network analysis techniques t
...
 
 The documentation is deployed on the branch `gh-pages` who's updation has been made automatic by the a [bash script](./ext/docs_auto_deploy.sh) which otherwise requires to run `make html` in docs directory after every commit.
```
Comment (prasadtalasila) [inline README.md]: remove this line and also line-23.

File: README.md [hunk 3]
```diff
@@ -13,37 +13,39 @@ The objective of this project is to utilize social network analysis techniques t
...
 
 You can view the documentation hosted on `gh-pages` [here](http://prasadtalasila.github.io/IRCLogParser/).
 
 
+## Installation
+
+IRCLogParser depends on various third-party libraries which are handled by [setup.py](./setup.py). 
+Run the "install.sh" script in the root directory to install these dependencies and other OS level
+dependencies for the third-party libraries.
+
```
Comment (prasadtalasila) [inline README.md]:
```bash
$git clone https://github.com/prasadtalasila/IRCLogParser.git
$cd IRCLogParser
$bash script/install
```

File: README.md [hunk 4]
```diff
@@ -13,37 +13,39 @@ The objective of this project is to utilize social network analysis techniques t
...
+
 ## Testing
 
 Presently, IRCLogParser has various end-to-end implemented which reside in the [test directory](./test/).
-To run the tests locally on your machine, run `python -m unittest -v tests` in the test directory.
+To run the all the tests locally on your machine, run `python -m unittest -v discover -s test/` in the root directory.
```
Comment (prasadtalasila) [inline README.md]: In the next two lines, separate the commands into code snippets.

Ex: To run the all the tests locally on your machine, run
```bash
python -m unittest -v discover -s test/
```
Placing the commands separately reduces errors in copy-paste process.

File: README.md [hunk 5]
```diff
@@ -13,37 +13,39 @@ The objective of this project is to utilize social network analysis techniques t
...
+To run a particular test, execute `python -m unittest -v location_to_test` in the root directory.
 
+Running all the unit & integration tests require 3-5 minutes.
 
-## Installation
+## Usage
+
+The library's working has been modularised into many modules namely input, analysis, output, visualisation and validation. The [sample.py file](./sample.py) very comprehensively presents how one can use IRCLogParser for parsing and analysis.
```
Comment (prasadtalasila) [inline README.md]: Please provide a shell command below this line.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]
@kaushik-rohit 
A few edits in the README.md; travis config looks good.

**Review by prasadtalasila — APPROVED** [review]

**Pull request merged**